common/ledger/blockledger/fileledger/factory_test.go

```diff
@@ -222,15 +222,15 @@ func TestRemove(t *testing.T) {
 		err = f.Remove("foo")
 		require.EqualError(t, err, "oogie")
 
-		dest := filepath.Join(dir, "filerepo", "remove", "foo.remove")
+		dest := filepath.Join(dir, "pendingops", "remove", "foo.remove")
 		_, err = os.Stat(dest)
 		require.NoError(t, err, "Expected foo.remove to exist")
 	})
 
-	t.Run("saving to file repo fails", func(t *testing.T) {
+	t.Run("saving to pending ops fails", func(t *testing.T) {
```

Comment:
Super nit: this didn't need to change and confuses things slightly but it doesn't matter. :) We're still using the filerepo as the underlying implementation (and that's what's failing to save for this test), it's just the public naming of the folder that changed.